plugins/www/Optimove.js

```diff
@@ -17,6 +17,7 @@ function init() {
 
 function resume() {
   setContext();
+  checkIfPendingDDLExists().then(null, (errMsg) => { console.error(errMsg); });
```

Comment:
why this is checked in `resume` vs for push we check it after handler is set. Any particular reason?

in general checking for this pending stuff is a total mess. Ideally would be better to have some event sync which would deliver a message after all is ready. We do that in Optimove Flutter and Kumulos react native i think. Not proposing changing it now

Reply:
I changed it to be like with the pending push